internal/compute/digitalocean_test.go

```
@@ -89,9 +39,12 @@
 		assert.NotNil(t, keyService)
 	})
 
-	t.Run("NewDOClient", func(t *testing.T) {
-		client := NewDOClient("test-token")
-		assert.NotNil(t, client)
+	t.Run("NewProvider", func(t *testing.T) {
+		os.Setenv("DIGITALOCEAN_TOKEN", "test-token")
```

golangci-lint (1.64.8)

Error return value of `os.Setenv` is not checked

(errcheck)